src/SLCore.Listeners.UnitTests/GlobalUsings.cs

```diff
@@ -0,0 +1,22 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+global using SonarLint.VisualStudio.SLCore.Listeners.Implementation;
+global using SonarLint.VisualStudio.TestInfrastructure;
```

Comment:
I assume this is an unrelated cleanup. It would make more sense to have this as a separate PR but since this PR is small enough already no need to separate it.

Reply:
Clean as You Create PRs